controllers/arenas_controller.js

```diff
@@ -38,7 +38,7 @@ exports.arena_details = async(req,resp)=>{
 }
 
 
-exports.arenas_details = async(req,resp)=>{
+exports.arenas_detail = async(req,resp)=>{
```

Comment:
why is this changes?

Reply:
beacause there was an error in naming convention